.pre-commit-config.yaml

```
@@ -1,10 +1,8 @@
 repos:
-  - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.86.0
+  - repo: https://github.com/pagopa/dx
+    rev: pre_commit_scripts@0.0.1
     hooks:
-      - id: terraform_fmt
-      - id: terraform_docs
-      - id: terraform_validate
-        args:
-          - --args=-json
-          - --args=-no-color
\ No newline at end of file
+      - id: lock_modules
+        exclude: ^.*/(_modules|\.terraform)(/.*)?$
```

I see that the pre-commit is also creating lock files in modules folders.
I think "*/modules/*" folders can be discarded cause evaluated at the root module

```suggestion
        exclude: ^.*/(modules|_modules|\.terraform)(/.*)?$
```

---

Yes, right I would remove modules from files

---

Those should fix the issue. Can you check, please?
- https://github.com/pagopa/io-auth-n-identity-domain/pull/181/commits/888ad7a30e92ad337f2760ddcea78192a87dd9a9: add `modules` into exclude
- https://github.com/pagopa/io-auth-n-identity-domain/pull/181/commits/7646cbbf268ca8b0a28e93f6ffa1e63b0baaa740: remove `modules` from files
- https://github.com/pagopa/io-auth-n-identity-domain/pull/181/commits/b1b3e149a294297cfbffff8ea1fbd2bfaf266e99: (manually) delete `tfmodules.lock.json` files from `infra/modules` folders